circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 4.0.0
+
+deployment:
+  production:
+    branch: master
+    commands:
+      - npm run deploy
```

Comment:
I don't think this is a script that `npm` knows about. As per the README, we `$(npm bin)/gulp deploy` to deploy.